src/component/actionbar/index.tsx

```diff
@@ -26,7 +23,7 @@ interface ActionBarProps extends DefaultActionBarProps {
 function ActionBar(props: ActionBarProps) {
   const { ventureId, currentTimeline, user, timelines, ...rest } = props;
   const { handleSubmit, register, reset } = useForm();
-  const token = useGetToken();
+  const { token } = useContext(AuthContext);
```

Comment:
An example of using the token from context. Still need to confirm behavior on expired tokens.